EIPS/eip-modexp-limit.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: nnnn
```

Comment:
```suggestion
eip: 7823
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.